src/main/java/org/opensearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -156,10 +156,7 @@
 import org.opensearch.ad.transport.handler.AnomalyIndexHandler;
 import org.opensearch.ad.transport.handler.AnomalyResultBulkIndexHandler;
 import org.opensearch.ad.transport.handler.MultiEntityResultHandler;
-import org.opensearch.ad.util.ClientUtil;
-import org.opensearch.ad.util.DiscoveryNodeFilterer;
-import org.opensearch.ad.util.IndexUtils;
-import org.opensearch.ad.util.Throttler;
+import org.opensearch.ad.util.*;
```

Comment:
minor: should we avoid using import *?

Reply:
Yes, will fix in other PRs since 1.3.8 is due today.